features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,7 @@
+Given("I visit the {string} page") do |string|
```

Comment:
do you use the string for something here?

Reply:
removed